inz-demo/internetnz/Issues/nethuiTicket/description.xml

```diff
@@ -0,0 +1,123 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo INZ Nethui Ticket</en>
+		<nl>Demo INZ Nethui Ticket</nl>
+	</Name>
+	<ShortName>
+		<en>Demo INZ Nethui Ticket</en>
+		<nl></nl>
+	</ShortName>
+	<SchemeManager>inz-demo</SchemeManager>
+	<IssuerID>internetnz</IssuerID>
+	<CredentialID>nethuiTicket</CredentialID>
+	<Description>
+		<en>INZ Nethui Ticket</en>
+		<nl></nl>
+	</Description>
+	<IssueURL>
+		<en></en>
+		<nl></nl>
+	</IssueURL>
+	<Attributes>
+		<Attribute id="id">
+			<Name>
+				<en>Ticket number</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Ticket number</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="event">
+			<Name>
+				<en>Event</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Event</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="eventDate">
+			<Name>
+				<en>Event date</en>
```

Comment:
What if the event is across many dates?

Reply:
Could be event start date?

Reply:
Do I keep event date info?

Reply:
Perhaps turn this into a "Valid until" date

Reply:
Maybe you can buy a 1 day pass, or a pass to all the events. A one day pass might need a valid from and until date? So example might be I dont want to go to the online event, but I want a 1 day pass to attend the in-person networking event on the 2nd day of Nethui.

Reply:
I think `eventDate` still is more clear.
```
1 day = 13/10/2020
or
1 day = 14/10/2020 (only second day)
2 days = 13 and 14/10/2020
or
2 days = 13, 14/10/2020
One week = 13-20/10/2020
```

Or any other standard that we define.

Reply:
Sure, happy to go with that given it's a PoC.